core/swift3Action/spm-build/Whisk.swift

```diff
@@ -94,9 +94,13 @@ class Whisk {
         let request = HTTP.request(requestOptions) { response in
             if response != nil {
                 do {
-                    let responseString = try response!.readString()!
-                    print(responseString)
-                    callback(["response": responseString])
+                    // this is odd, but that's just how KituraNet has you get
+                    // the response as NSData
+                    let jsonData = NSMutableData()
+                    try response!.readAllData(into: jsonData)
+
+                    let resp = try NSJSONSerialization.jsonObject(with: jsonData, options: [])
+                    callback(["activation": resp])
```

Comment:
since the response is a json object (and usually the activation record if no error), is it necessary to nest it at all?